.vale/styles/style_guide/British.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Suggestion: style_guide.British.yml
+# Checks for American  spelling
+#
+extends: substitution
+message: 'Use the American spelling "%s" instead of the British "%s".'
+link: https://about.gitlab.com/handbook/communication/#top-misused-terms
+level: suggestion
+ignorecase: true
+swap:
+  analyse: analyze
+  behaviour: behavior
+  centre: center
+  colour: color
+  dependant: dependent
```

Comment:
huh, I was taught dependent as being the adjective and dependant as the noun ("this is thing is dependent on that thing" but "he is my dependant"). Good to to know the existence no longer exists.

Reply:
Same